src/components/FeeSelector/shared.tsx

```diff
@@ -1,7 +1,7 @@
 import { Trans } from '@lingui/macro'
 import { FeeAmount } from '@uniswap/v3-sdk'
 import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from 'constants/chains'
-import { ReactNode } from 'react'
+import type { ReactNode } from 'react'
```

Comment:
revert this line?

Reply:
i feel like we don't use this syntax elsewhere so probably prefer to avoid fwiw